flink-runtime/src/test/java/org/apache/flink/runtime/filecache/FileCacheDirectoriesTest.java

```diff
@@ -206,4 +200,49 @@ public void testDirectoryCleanUp() throws Exception {
 			}
 		}
 	}
+
+	@Test
+	public void testCompression() throws IOException {
```

Comment:
If we move the zip and expand methods to `FileUtils`, then this test should be moved as well.